docs/resources/cluster_v2.md

```diff
@@ -236,6 +236,96 @@ The `<AUTH_CONFIG_SECRET_NAME>` represents a generic kubernetes secret which con
 
 Many registries may be specified in the `rke_config`s `registries` section, however the `system-default-registry` from which core system images are pulled is always denoted via the `system-default-registry` key of the `machine_selector_config` or the `machine_global_config`. For more information on private registries, please refer to [the Rancher documentation](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/global-default-private-registry#setting-a-private-registry-with-credentials-when-deploying-a-cluster) 
 
+### Creating Rancher V2 cluster with cluster agent customization. For Rancher v2.7.5 and above.
+
+```hcl
+resource "rancher2_cluster_v2" "foo" {
+  name = "foo"
+  enable_network_policy = false
+  rke_config {
+    machine_pools {
```

Comment:
This resource don't need the kubernetes_version arg ? 

This doc says that is required.